### PR TITLE
Release v0.28.0

### PR DESCRIPTION
**Pull request by jcheng-splunk**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,20 @@
...
 
+## v0.28.0
+
+This Splunk OpenTelemetry Collector release includes changes from the [opentelemetry-collector v0.28.0](https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.28.0) and the [opentelemetry-collector-contrib v0.28.0](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.28.0) releases.
+
+### 💡 Enhancements 💡
+
+- Initial puppet module for linux (#405)
+- Add YAML config source (#402)
+- Add file support to include config source (#397, #419)
```
Comment (rmfitzpatrick) [inline CHANGELOG.md:13]:
```suggestion
- Add `include` config source (#419, #402, #397)
```

**Comment by jrcamp** [comment]
@rmfitzpatrick @pjanotti ptal

File: go.mod [hunk 2]
```diff
@@ -3,67 +3,67 @@ module github.com/signalfx/splunk-otel-collector
...
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/sapmreceiver v0.28.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/signalfxreceiver v0.28.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/simpleprometheusreceiver v0.28.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/splunkhecreceiver v0.28.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/statsdreceiver v0.28.0
 	github.com/openzipkin/zipkin-go v0.2.5
 	github.com/signalfx/defaults v1.2.2-0.20180531161417-70562fe60657
 	github.com/signalfx/golib/v3 v3.3.33
-	github.com/signalfx/signalfx-agent v1.0.1-0.20210503202607-8862b3ba9c0d
+	github.com/signalfx/signalfx-agent v1.0.1-0.20210325135021-a18ea9d77b40
```
Comment (rmfitzpatrick) [inline go.mod:42]:
```suggestion
       github.com/signalfx/signalfx-agent v1.0.1-0.20210610210307-b3be6d1b7fa1
```

**Review by rmfitzpatrick — APPROVED** [review]
Verified integration and end-to-end tests

**Pull request merged**